README.md

```diff
@@ -196,7 +197,7 @@ const dbName = 'myproject';
 // Use connect method to connect to the server
 MongoClient.connect(url, function(err, client) {
```

Comment:
Would we want to change the example to use the `MongoClient` constructor instead? (Just to get people in the habit of doing that since if they want to listen to events from the very beginning, like SDAM events, they will need to register the handlers before connecting)

Reply:
SGTM